barman/retention_policies.py

```diff
@@ -185,6 +214,16 @@ def _backup_report(self, source):
                 i = i + 1
             else:
                 report[bid] = BackupInfo.NONE
+
+            if backups[bid].has_children:
+                status = report[bid]
+                if status in (BackupInfo.KEEP_STANDALONE, BackupInfo.KEEP_FULL):
+                    status = BackupInfo.VALID
```

Comment:
Maybe we want to push this "use VALID if status in X or Y" from here to `propagate_retention_status_to_children`?

I mean, we perform this logic twice in this file, related to the same thing.